x-pack/plugins/maps/common/constants.ts

```diff
@@ -97,6 +97,7 @@ export const ZOOM_PRECISION = 2;
 export const DEFAULT_MAX_RESULT_WINDOW = 10000;
 export const DEFAULT_MAX_INNER_RESULT_WINDOW = 100;
 export const DEFAULT_MAX_BUCKETS_LIMIT = 65535;
+export const DEFAULT_TERM_JOIN_SIZE = 250; // why 250 - there are 250 countries in EMS world counties layer so using default that works for that example
```

Comment:
discussed offline. woudl consider actually just putting in a "placeholder" instead that defaults to the existing max_bucket_limit. It would remove the need for a user to have to explicitly configure this setting each time they add a join.